HashRegistrarSimplified.sol

```diff
@@ -451,7 +451,8 @@ contract Registrar {
         if(address(h.deed) != 0) {
             // Reward the discoverer with 50% of the deed
             // The previous owner gets 50%
-            h.deed.setBalance(h.deed.value()/2);
+            h.value = max(h.value, minPrice);
```

Comment:
When would `minPrice` be greater than `h.value` ? 
And if so, can it be guaranteed that `h.deed.balance` is also larger than `h.value/2` ? 
Because if not, then `setBalance` could fail, and it wouldn't be possible to invalidate the name....